bin/tester/index.js

```diff
@@ -483,17 +483,19 @@ export async function runTest({
       }
     }
 
-    if (!publishOnly) {
-      log.info(dedent`
-      Started Build ${buildNumber} (${pluralize(componentCount, 'component')}, ${pluralize(
-        specCount,
-        'story'
-      )}, ${pluralize(snapshotCount, 'snapshot')}).
+    const onlineHint =
+      buildNumber === 1 ? `Continue setup at ${setupUrl}` : `View it online at ${exitUrl}`;
```

Comment:
I wonder if we should try to be smarter here? On the backend we have some extra logic to deal with the not completely uncommon case of the user having a few builds fail as they get things setup: https://github.com/chromaui/chromatic/blob/release/services/index/model/Build.js#L727-L733

Perhaps we should check if `build.autoAcceptChanges=true` but we didn't ask for it to be?

Reply:
Excellent suggestion, love it! Done.